### PR TITLE
Add constain->contain

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -4712,6 +4712,7 @@ consrtuctor->constructor
 consrtuctors->constructors
 consrtucts->constructs
 consructor->constructor
+constain->contain
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: also "constrain"

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: done,  ddf5492

**Comment by luzpaz** [comment]
Done, ready for merge

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -4731,7 +4731,9 @@ consrtuctor->constructor
 consrtuctors->constructors
 consrtucts->constructs
 consructor->constructor
+constain->constrain, contain,
 constained->constrained
+constaining->constraining
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can we have containing, in fact please can you add contain* as a suggestion to all these typos.

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 3033264

**Comment by luzpaz** [comment]
Ready for merge

**Pull request merged**